supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -217,6 +217,7 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
         .setResources(resources)
         .setActions(sortedActions.asJava)
         .setEnvironment(environment)
+        .setTimeout(createPipelineParameters.pipelineTimeout)
```

Comment:
if `createPipelineParameters.pipelineTimeout` were an `Option[FiniteDuration]`, this is where you could do something like:
```
createPipelineParameters.pipelineTimeout foreach {  
  timeout => setTimeout(timeout.toSeconds.toString() + "s") 
}
```